vllm/model_executor/openvino_model_loader.py

```diff
@@ -599,10 +599,12 @@ def get_model(model_config: ModelConfig,
         else:
             print(f'[ INFO ] OpenVINO IR is avaialble for provided model id {model_config.model}. '
                   'This IR will be used for inference as-is, all possible options that may affect model conversion are ignored.')
+        load_in_8bit = None if os.environ.get('VLLM_OPENVINO_ENABLE_QUANTIZED_WEIGHTS', '0') == '1' else False
```

Comment:
please, list in documentation https://github.com/ilya-lavrenov/vllm/blob/openvino-model-executor/use_with_openvino.md